src/json-schema/partial/map_state.json

```diff
@@ -49,6 +49,219 @@
                                 "States"
                             ]
                         },
+                        "ItemReader": {
+                            "type": "object",
+                            "description": "Specifies the location of the dataset from which the Map state reads its input data.",
+                            "properties": {
+                                "ReaderConfig": {
+                                    "type": "object",
+                                    "description": "A JSON object that specifies where to read the items instead of from the effective input.",
+                                    "properties": {
+                                        "InputType": {
+                                            "type": "string",
```

Comment:
Can we use an enum or oneOf to limit the accepted values? This would be similar to choice states.

Reply:
Yeah.